contentcuration/contentcuration/frontend/shared/app.js

```diff
@@ -43,6 +46,8 @@ Vue.use(KThemePlugin);
 // Register global components
 Vue.component('ActionLink', ActionLink);
 
+i18nSetup();
```

Comment:
This is now an asynchronous function, due to the async loading in the polyfill. We should put this into the `startApp` function below and await its resolution before allowing Vue to start.